tesseract_state_solver/src/kdl_state_solver.cpp

```diff
@@ -109,7 +112,13 @@ void KDLStateSolver::setState(const std::vector<std::string>& joint_names,
   calculateTransforms(current_state_, kdl_jnt_array_, data_.tree.getRootSegment(), Eigen::Isometry3d::Identity());
 }
 
-SceneState KDLStateSolver::getState(const Eigen::Ref<const Eigen::VectorXd>& joint_values) const
+void KDLStateSolver::setState(const tesseract_common::TransformMap& /*floating_joint_values*/)
+{
+  throw std::runtime_error("KDLStateSolver, not supported!");
```

Comment:
Why are floating joints not supported for the KDL state solver?

Reply:
The KDLStateSolver for the most part is only leveraged for validating the OFKT state solver so functionality like this is not needed. Once I get around to update the joint group class to leverage OFKT this state solver will be moved to the testing directory only to be used for testing OFKT State solver.

Reply:
It looks like you need to rebuild the tree which is what I what I thought. I think this why I did not go through the trouble before and making KDL inherit from the mutable state solver. In the unit tests I modify the scene graph and the construct the kdl state solver each time.